secure/key-manager/src/tests.rs

```diff
@@ -100,30 +106,31 @@ impl Node {
             libra,
             storage,
             _storage_service: storage_service,
-            _time: time,
+            time,
         }
     }
 
-    // TODO(davidiw): Notify external upon reconfigure and update epoch via ledger_info and also add
-    // next validator set if appropriate...
     fn execute_and_commit(&self, mut block: Vec<Transaction>) {
+        // 1) Update the clock for potential reconfigurations
+        self.time.increment();
+        // Clock is supposed to be in microseconds
+        let clock = self.time.now() * 1_000_000;
```

Comment:
you probably can put the clock in the LI to make it more real

Reply:
Hmm.... lemme think about that.